cherrypy/tutorial/tutorial.conf

```diff
@@ -1,4 +1,4 @@
 [global]
-server.socket_host = "127.0.0.1"
+server.socket_host = "0.0.0.0"

```

Comment:
I don't think it's wise to change the config for the tutorial not to use best practices for secure development.